daostack/common-web
# New common feed item design #1329

https://github.com/daostack/common-web/issues/1329

### What was changed?
- Added new design for desktop and mobile feed card
- Added FeedItemDetailsPreview
- Added logic for expand card

### How to test?
- Open Feed in new design
- Open Desktop 
- Extend card
- Open Mobile
- Extend Card
- Open MobileChat for FeedCard
- Click See more
